src/commands/heroku.rs

```diff
@@ -390,6 +398,177 @@ pub fn restart_app(ctx: &mut Context, msg: &Message, mut args: Args) -> CommandR
     Ok(())
 }
 
+#[command]
+#[num_args(3)]
+pub fn deploy_app(ctx: &mut Context, msg: &Message, mut args: Args) -> CommandResult {
+    let app_name = args
+        .single::<String>()
+        .expect("You must include an app name");
+
+    let app_version = args
+        .single::<String>()
+        .expect("You must include an app version");
+
+    let git_ref = args
+        .single::<String>()
+        .expect("You must include a git ref to deploy");
+
+    let github_client = Github::new(bot_config(ctx).github_token.to_string());
+
+    if github_client.is_err() {
+        msg.reply(
+            &ctx,
+            format!(
+                "An error occured when trying to contact GitHub with your GitHub token:\n{:?}",
+                github_client.err()
+            ),
+        )?;
+
+        return Ok(());
+    }
+
+    let github_response = github_client
+        .unwrap()
+        .get()
+        .custom_endpoint(&github_ref_endpoint(bot_config(ctx), git_ref))
+        .execute::<Value>();
```

Comment:
I thought it would be easier to have it on hand if we added more GitHub functionality in the future, but I'm not opposed to writing our own function for just this endpoint until we need more. I'll get that in!

Reply:
Yeah let's write our own function here.

Reply:
Done!